.golangci.yaml

```diff
@@ -11,6 +11,10 @@ issues:
     - path: '(.+)_test\.go'
       linters:
         - govet
+    - linters: [revive]
+      text: var-naming  # do not demand that `Id` always be spelled `ID`; messes with autogenerated code
```

Comment:
I disliked it pretty strongly actually :), because it's just a bunch of exceptions to CamelCase.

Still, I tried to keep it going because it's The Go Way, and peppered our .yaml with custom renames (e.g. JSON `entity_id` should explicitly become `EntityID` in Go, not `EntityId`). Those exceptions are still around. But such renames are only supported for externally-facing types, not the autogenerated function names and server-internal types. So it's a necessary sacrifice on the altar of codegen.